src/main/java/org/gridsuite/filter/server/FilterController.java

```diff
@@ -55,16 +55,18 @@ public FilterController(FilterService service) {
             .orElse(ResponseEntity.notFound().build());
     }
 
-    @PostMapping(value = "filters/", consumes = MediaType.APPLICATION_JSON_VALUE)
+    @PostMapping(value = "/filters/{id}", consumes = MediaType.APPLICATION_JSON_VALUE)
     @Operation(summary = "Create a filter")
     @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The filter has been successfully created")})
-    public ResponseEntity<AbstractFilter> createFilter(@RequestBody(required = true) AbstractFilter filter) {
+    public ResponseEntity<AbstractFilter> createFilter(@PathVariable("id") UUID filterId,
+                                                       @RequestBody AbstractFilter filter) {
+        filter.setId(filterId);
```

Comment:
I really dont like that an element object in directory-server have the same Uuid than a filter object in filter-server...